src/__tests__/extensions/sessionrecording.js

```diff
@@ -70,14 +80,22 @@ describe('SessionRecording', () => {
             _emit({ event: 2 })
 
             expect(given.posthog.capture).toHaveBeenCalledTimes(2)
-            expect(given.posthog.capture).toHaveBeenCalledWith('$snapshot', {
-                $session_id: 'sid',
-                $snapshot_data: { event: 1 },
-            })
-            expect(given.posthog.capture).toHaveBeenCalledWith('$snapshot', {
-                $session_id: 'sid',
-                $snapshot_data: { event: 2 },
-            })
+            expect(given.posthog.capture).toHaveBeenCalledWith(
+                '$snapshot',
+                {
+                    $session_id: 'sid',
+                    $snapshot_data: { event: 1 },
+                },
+                { method: 'POST', transport: 'XHR', endpoint: '/e/' }
```

Comment:
Shouldn't `$snapshot`s be sent to `/ses/`?

Reply:
/decide endpoint decides where it should go (for backwards compatibility with new posthog-js and old posthog app).

In this test /decide did not respond with the /ses/ endpoint